src/components/InventoryManager/ListingSteps/PricesStep.js

```diff
@@ -0,0 +1,226 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import ListingInputField from '../ListingInputField';
+
+const useStyles = makeStyles({
+  heading: {
+    fontFamily: 'Work Sans',
+    fontWeight: '700',
+    textDecoration: '#53AA48 solid underline 2px',
+    textUnderlinePosition: 'under',
+    fontSize: '1.4rem',
+  },
+  subheading: {
+    fontFamily: 'Work Sans',
+    fontWeight: '600',
+    fontSize: '1rem',
+  },
+  title: {
+    fontFamily: 'Work Sans',
+    fontSize: '2.3rem',
+    fontWeight: '400',
+  },
+  text: {
+    fontFamily: 'Work Sans',
+    fontSize: '1rem',
+    fontWeight: '400',
+  },
+  labels: {
+    fontFamily: 'Work Sans',
+    fontWeight: '600',
+    fontSize: '1.05rem',
+  },
+  unitLabel: {
+    fontFamily: 'Work Sans',
+    fontSize: '1rem',
+    fontWeight: '400',
+    paddingLeft: '.5rem',
+  },
+});
+
+export default function PricesStep({
+  setListingRecord, listingRecord,
+}) {
+  const classes = useStyles();
+  const questions = [
+    '1. What is the standard price of your produce?',
+    '2. What is the agency price of your produce?',
+  ];
+  const onChangePrice = (e, newValue, target) => {
+    setListingRecord({ ...listingRecord, [target]: newValue });
+  };
+  const validPrices = `${listingRecord['has master units'] ? 'Master Unit/' : ''}Pallet${listingRecord['has master pallets'] ? '/Master Pallet' : ''}`;
+  const getPrices = (basePrice) => {
+    const prices = {};
+    if (listingRecord['has master units']) {
+      prices.masterUnitPrice = listingRecord['grouped produce type per master unit'] * basePrice;
+    } else {
+      prices.masterUnitPrice = 0;
+    }
+    prices.palletPrice = prices.masterUnitPrice * listingRecord['master units per pallet'] + basePrice * listingRecord['grouped produce type per pallet'];
```

Comment:
Just following up on this again! Shouldn't palletPrice be standard price per grouped produce type * grouped produce type per pallet? So assuming basePrice is the same as standard price/agency price per grouped produce type, it's just one multiplication right?